src/react-components/avatar-editor.js

```diff
@@ -47,6 +47,53 @@ const fetchAvatar = async avatarId => {
   return avatars[0];
 };
 
+// GLTFLoader plugin for splitting glTF and bin from glb.
+class GLTFBinarySplitterPlugin {
+  constructor(parser, inputFiles) {
+    this.parser = parser;
+    this.inputFiles = inputFiles;
+  }
+
+  beforeRoot() {
+    const parser = this.parser;
+    const { body } = parser.extensions.KHR_binary_glTF;
+    const content = JSON.stringify(ensureAvatarMaterial(parser.json));
+
+    // Inject hubs components on upload. Used to create base avatar
```

Comment:
I think you can just get rid of this commented code now. This predates the Blender extension and was an easy way for us to create some of the initial avatars, but we don't need it anymore.

Reply:
Done, thanks